### DISCUSSION TITLE
YAML library

### DISCUSSION THREAD

**Discussion by tobozo** [earlephilhower/arduino-pico#1063] [General]
hi and thanks for this awesome project :+1: 

I found this [issue](https://github.com/earlephilhower/arduino-pico/issues/978) and thought posting here may be of some interest for the community.

The [YAMLDuino library](https://github.com/tobozo/YAMLDuino) has support for arduino-pico platform.
What started as a leaky YAML<=>JSON converter can now be used along with ArduinoJSON accessors to manipulate YAML, and comes with a gettext() provider for localization.

https://github.com/tobozo/YAMLDuino

any form of feedback is welcome :wink:

**Comment by earlephilhower** [comment]
Did you register w/the Arduino library system?  That way folks can find your lib right from the IDE.

**Reply by tobozo** [reply]
yup it's been in the Arduino library registry for a couple of months now, also in Platformio lib registry
